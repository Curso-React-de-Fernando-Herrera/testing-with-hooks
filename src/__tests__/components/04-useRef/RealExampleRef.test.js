import React from 'react'
import { RealExampleRef } from "components/04-useRef/RealExampleRef"
import { shallow } from "enzyme"

describe('RealexampleRef component', () => {
  test('should be render', () => {
    const wrapper = shallow( <RealExampleRef />)
    expect(wrapper).toMatchSnapshot()
  })

  test('should show MulticustomHooks', () => {
    const wrapper = shallow( <RealExampleRef /> )

    const button = wrapper.find('button')
    button.simulate('click')

    expect( wrapper.find('MultipleCustomHooks').exists() ).toBe( true )
  })
})