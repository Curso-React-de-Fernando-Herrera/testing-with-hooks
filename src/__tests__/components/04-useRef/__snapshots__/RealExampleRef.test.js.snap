// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RealexampleRef component should be render 1`] = `
<div>
  <h1>
    RealExampleRef
  </h1>
  <hr />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
    Show/Hide
  </button>
</div>
`;
